src/plots/VolumeVTK9/PyVolumeAttributes.C

```diff
@@ -3456,44 +3456,50 @@ PyVolumeAttributes_getattr(PyObject *self, char *name)
     if(strcmp(name, "materialProperties") == 0)
         return VolumeAttributes_GetMaterialProperties(self, NULL);
 
-    // Try and handle legacy fields
+#include <visit-config.h>
```

Comment:
Are you ok with this `#include` directive here in the *middle* of the file? I suspect its the result of a `.code` file organization and if so, thats probably ok.